.github/workflows/pr-quality.yml

```
@@ -35,13 +35,13 @@ jobs:
       - name: Run comprehensive quality analysis
         id: quality
         run: |
-          # Use our enhanced quality system
-          echo "🔍 Running quality analysis..."
-          npm run quality:check > quality-full.txt 2>&1 || true
+          # Use our validation pipeline
+          echo "🔍 Running validation pipeline..."
+          npm run validate:pr:fast > quality-full.txt 2>&1 || true
           
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)